documentation/pom.xml

```diff
@@ -20,7 +20,7 @@
     <packaging>jar</packaging>
 
     <name>JSR-380 TCK Documentation</name>
-    <description>&gt;Bean Validation TCK Documentation</description>
+    <description>&gt;Jakarta Bean Validation TCK Documentation</description>
```

Comment:
Do you know what there is a greater than?

Reply:
No, I think it was just a mistake. The other artifacts don't have one and it's weird so I removed it.